packages/mobile/local_core/generated/graphql.tsx

```diff
@@ -9,7 +9,9 @@ export type Scalars = {
   Boolean: boolean;
   Int: number;
   Float: number;
+  /** Date custom scalar type */
   Date: any;
+  /** The `Upload` scalar type represents a file upload. */
```

Comment:
I don't know if these comments are necessary. If there are please change to one-line comment with `//` instead of `/** */`

Reply:
This line is from the generated file from graphql